src/test/java/com/wikia/webdriver/testcases/discussions/CategoriesTests.java

```diff
@@ -86,7 +86,7 @@ private void deleteCategoriesMobile() {
   }
 
   private void deleteCategories(String wikiName) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "deleteCategories" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)